### ISSUE TITLE
Font axis (ital, wght) are wrong

### ISSUE THREAD

**Issue by bart** [nuxt-community/google-fonts-module#30]
It looks like the Google Fonts Module is handling font axis in the wrong way. I'll give you an example:

```
'Roboto Condensed': {
  wght: [300, 400, 700],
  ital: [400],
},
```

This families definition generates the following path: https://fonts.googleapis.com/css2?family=Roboto%20Condensed:ital,wght@0,400;1,300;1,400;1,700&display=swap

wght and ital attributes are correctly assigned. BUT compared to Google Fonts Selection it should generate https://fonts.googleapis.com/css2?family=Roboto%20Condensed:ital,wght@0,300;0,400;0,700;1,400&display=swap

In my opinion this is a Google Fonts API issue. Seems like they flipped wght and ital attribute internally. If you open the given CSS file Roboto Condensed Bold is just generated for the italic style but not (like expected) for the normal one.

**Comment by ricardogobbosouza** [comment]
Hi @bart 
Fixed on [v1.1.5](https://github.com/nuxt-community/google-fonts-module/releases/tag/v1.1.5)

**Comment by bart** [comment]
As I said, I think this is more a Google Fonts issue than a plugin issue. But maybe "solving" this in your plugin prevents other people from having the same question like I had. On the other hand if they just copy it blind from Google Fonts frontend they might expect the exact same issue because then it's double flipped :)

Thanks for your effort!
